packages/shared/src/components/comments/MainComment.tsx

```diff
@@ -94,7 +96,7 @@ export default function MainComment({
           comment={comment}
           parentId={comment.id}
           className={{
-            container: 'border-b',
+            container: comment.children?.edges?.length > 0 && 'border-b',
```

Comment:
We should only add bottom border if there are children. Otherwise we end up with double borders.